packages/web/integration-tests/tests/errors/views/script1.min.js

```diff
@@ -0,0 +1,5 @@
+document.getElementById("button1").addEventListener("click",function(){throw new Error("Error from script1.js")});
+(function() {
```

Comment:
Probably should also wrap the content here in try catch block (see issue above about document.currentScript not being set in modules, likely could also end up as an issue if someone does something like `jQuery.getScript`)